jdtls.ext/com.microsoft.jdtls.ext.core/src/com/microsoft/jdtls/ext/core/ProjectCommand.java

```diff
@@ -170,24 +156,39 @@ public static ExportResult exportJar(List<Object> arguments, IProgressMonitor mo
             Set<String> directories = new HashSet<>();
             for (Classpath classpath : classpaths) {
                 if (monitor.isCanceled()) {
-                    return new ExportResult(false, "User cancelled");
+                    return false;
                 }
                 if (classpath.isArtifact) {
-                    writeArchive(new ZipFile(classpath.source), /* areDirectoryEntriesIncluded = */true,
-                        /* isCompressed = */true, target, directories, monitor);
+                    MultiStatus resultStatus = writeArchive(new ZipFile(classpath.source),
+                        /* areDirectoryEntriesIncluded = */true, /* isCompressed = */true, target, directories, monitor);
+                    int severity = resultStatus.getSeverity();
+                    if (severity == IStatus.OK) {
+                        java.nio.file.Path path = java.nio.file.Paths.get(classpath.source);
+                        reportExportJarMessage(taskLabel, IStatus.OK, "successfully export " + path.getFileName().toString());
```

Comment:
The message is not accurate.

Successfully added the file to the exported jar: path.getFileName().toString()